canonical/ubuntu-pro-client
# i18n (SC-1517)

## Why is this needed?

This PR adds translation support and delivers translations in a new separate package that is "Recommend"ed by ubuntu-advantage-tools.

## Test Steps
Set up a build chroot with the version of pkgbinarymangler that has ubuntu-pro-client-l10n in striptranslations.blacklist (there might be a better way to do this?)
```
sudo sbuild-launchpad-chroot create -s xenial -a amd64 -n pro-test-xenial-amd64

sudo schroot -c source:pro-test-xenial-amd64 -d / -u root -- sh -c 'echo "deb http://ppa.launchpad.net/orndorffgrant/pkgbinarymangler-ubuntu-pro-client-l10n/ubuntu xenial main" >> /etc/apt/sources.list.d/testppa.list'
sudo schroot -c source:pro-test-xenial-amd64 -d / -u root -- sh -c 'apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 74E59E9E'

sudo schroot -c source:pro-test-xenial-amd64 -d / -u root -- sh -c 'apt update'
sudo schroot -c source:pro-test-xenial-amd64 -d / -u root -- sh -c 'apt install pkgbinarymangler'

./tools/build.sh --chroot pro-test-xenial-amd64 xenial
./tools/test-in-lxd.sh xenial
LANGUAGE=pt_BR.UTF-8 pro security-status
```

You should see one line that is translated.


## Checklist


## Does this PR require extra reviews?
